views/components/Drawer.jsx

```diff
@@ -9,218 +8,206 @@ import ListItemIcon from '@material-ui/core/ListItemIcon'
+                    style={{
+                      fontWeight: 'bold',
+                    }}
+                  >
+                    {props.totalDays}
+                  </Typography>
+                </td>
+              </tr>
+              <tr>
+                <td style={{ width: '30%' }}>
+                  <Typography
+                    noWrap={true}
+                    style={{
+                      color: 'rgba(0, 0, 0, 0.75)',
+                    }}
+                  >
+                    {'studies'}
+                  </Typography>
+                </td>
+                <td style={{ width: '30%' }}>
+                  <Typography
+                    noWrap={true}
+                    style={{
+                      color: 'rgba(0, 0, 0, 0.75)',
+                    }}
+                  >
+                    {'participants'}
+                  </Typography>
+                </td>
+                <td style={{ width: '30%' }}>
+                  <Typography
+                    noWrap={true}
+                    style={{
+                      color: 'rgba(0, 0, 0, 0.75)',
+                    }}
+                  >
+                    {'days'}
+                  </Typography>
+                </td>
+              </tr>
+            </tbody>
+          </table>
         </div>
-        <Divider />
-        <List dense={true}>
-          <ListItem button={true} component="a" href={routes.home}>
-            <ListItemIcon>
-              <Home style={{ color: colors.dark_sky_blue }} />
-            </ListItemIcon>
-            <ListItemText primary="Home" />
-          </ListItem>
-          <ListItem button={true} component="a" href={routes.charts}>
-            <ListItemIcon>
-              <ShowChart style={{ color: colors.dark_sky_blue }} />
-            </ListItemIcon>
-            <ListItemText primary="Charts" />
-          </ListItem>
-          <ListItem button={true} component="a" href={routes.configure}>
-            <ListItemIcon>
-              <ColorLens style={{ color: colors.dark_sky_blue }} />
-            </ListItemIcon>
-            <ListItemText primary="Configure" />
-          </ListItem>
-          <ListItem button={true} component="a" href={routes.userAccount}>
-            <ListItemIcon>
-              <Person style={{ color: colors.dark_sky_blue }} />
-            </ListItemIcon>
-            <ListItemText primary="Account" />
-          </ListItem>
-          {this.props.user.role == 'admin' ? (
-            <ListItem button={true} component="a" href={routes.admin}>
-              <ListItemIcon>
-                <Settings style={{ color: colors.dark_sky_blue }} />
-              </ListItemIcon>
-              <ListItemText primary="Admin" />
-            </ListItem>
-          ) : null}
-          <ListItem button={true} component="a" href={routes.logout}>
+      </div>
+      <Divider />
+      <List dense={true}>
+        <ListItem button={true} component={RouterLink} to={routes.home}>
```

Comment:
I think we still want these list items to be `li` elements, but we should render the RouterLinks inside the `li`s.